dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -219,6 +220,19 @@ private void initializeAdditionalFields(String include, User user) {
         }
     }
 
+    @GET
+    @Timed
+    @UnitOfWork(readOnly = true)
+    @RolesAllowed("admin")
+    @Operation(operationId = "listUsers",
+            description = "List all users",
+            responses = {@ApiResponse(responseCode = HttpStatus.SC_OK + "",
+                    content = @Content(mediaType = MediaType.APPLICATION_JSON, array = @ArraySchema(schema = @Schema(implementation = User.class))), description = "A list of all users")},
+            security = @SecurityRequirement(name = JWT_SECURITY_DEFINITION_NAME))
+    public List<User> listUsers() throws JsonProcessingException {
```

Comment:
This came in the review of the draft and has also come up in security review meetings, how we have to go to the DB to figure out the curators, so I put this in there, accessible only to admins.

This will not be invoked from the UI, as least not yet, so I punted on pagination and filtering, which I think is OK for now.

Reply:
Removed the endpoint; can discuss as part of another issue/PR.